modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -1,7 +1,7 @@
 = Using the occ Command
 :toc: macro
 :toclevels: 2
-:page-aliases: configuration/server/occ_app_command.adoc,go/admin-cli-upgrade.adoc
+:page-aliases: configuration/server/occ_app_command.adoc,go/admin-cli-upgrade.adoc,configuration/files/external_storage_configuration.adoc
```

Comment:
Just for my knowledge, what is the `go` in `configuration/server/occ_app_command.adoc,go` good for?

Reply:
That is for redirects from ownCloud core to the documentation. I don't have an example to hand, unfortunately.

Reply:
Maybe it reads `go/admin...` but it really looks very strange

Reply:
Just confirming that this redirect is correct, @mmattel